src/routes/AuthorityViewRoute/AuthorityViewRoute.js

```diff
@@ -26,6 +26,22 @@ import {
 
 import { AuthorityView } from '../../views';
 
+const checkCanLoadMarcSource = (isConsortiaEnv, isShared, isConsortiumDataLoaded, isAuthorityLoaded) => {
```

Comment:
It returns a boolean; just call it `canLoadMarcSource`. Functions named `checkFoo` or `validateFoo` should return null/undefined on success and throw an error on failure. Functions named like `isValidFoo` or `canLoadFoo` should return `true` on success and `false` on failure. 

Consider adding the description from the PR as a header comment for this function to answer _why_ these are the relevant conditions to evaluate when deciding if it is possible to load MARC source.